res/controllers/Traktor-Kontrol-S3-hid-scripts.js

```diff
@@ -0,0 +1,1694 @@
+///////////////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                                          //
+///////////////////////////////////////////////////////////////////////////////////
+/* global controller                                                             */
+/* global HIDDebug                                                               */
+/* global HIDPacket                                                              */
+/* global HIDController                                                          */
+/* jshint -W016                                                                  */
```

Comment:
```suggestion
```

We're using eslint now :D

Reply:
when I remove this I get:

  20:27  error  'HIDController' is not defined  no-undef
   173:9   error  'HIDDebug' is not defined       no-undef
   383:9   error  'HIDDebug' is not defined       no-undef
  1059:9   error  'HIDDebug' is not defined       no-undef
  1098:28  error  'HIDPacket' is not defined      no-undef
  1099:27  error  'HIDPacket' is not defined      no-undef
  1319:23  error  'HIDPacket' is not defined      no-undef
  1320:23  error  'HIDPacket' is not defined      no-undef
  1595:9   error  'HIDDebug' is not defined       no-undef
  1604:17  error  'HIDDebug' is not defined       no-undef
  1609:17  error  'HIDDebug' is not defined       no-undef
  1614:13  error  'controller' is not defined     no-undef
  1648:9   error  'controller' is not defined     no-undef
  1651:5   error  'HIDDebug' is not defined       no-undef
  1669:5   error  'HIDDebug' is not defined       no-undef

Reply:
ah, got it